jefferson/jffs2.py

```diff
@@ -180,7 +180,8 @@ def unpack(self, data):
             elif self.compr == JFFS2_COMPR_LZMA:
                 self.data = jffs2_lzma.decompress(node_data, self.dsize)
             elif self.compr == JFFS2_COMPR_LZO:
-                self.data = lzo.decompress(node_data, False, self.dsize)
+                compressor = LZOCompressor()
+                self.data = compressor.decompress(node_data)
```

Comment:
It is a static method, no need to instantiate:
```suggestion
                self.data = LZOCompressor.decompress(node_data)
```

Reply:
Fixed